python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -747,6 +757,8 @@ def run_transformer_int4_gpu_win(repo_id,
     from omegaconf import OmegaConf
     conf = OmegaConf.load(f'{current_dir}/config.yaml')
     today = date.today()
+    if 'exclude' in conf:
+        excludes = conf['exclude']
```

Comment:
is this correct? This is in `run_transformer_int4_gpu_win` not in `run_transformer_int4_gpu`?